version/version.go

```diff
@@ -17,3 +23,11 @@ var BuildNumber = "dev-build"
 func AsString() string {
 	return fmt.Sprintf("Branch: %s. Build id: %s. Commit: %s", GitBranch, BuildNumber, GitCommit)
 }
+
+func GetInfo() *Info {
```

Comment:
Since we're exposing this method to get version info, can `GitCommit`, `GitBranch` and `BuildNumber` be private? Or do they have to be public because we set them in bash script?

Reply:
The logic should be very simple. Version package provides a few exported variables and a function which concats them to string. Why do we need separate GetInfo function with variables put in structure? Just define structure where you need it - i.e. in healthcheck endpoint